app/routes/goal_routes.py

```diff
@@ -0,0 +1,105 @@
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, abort, make_response, request
+from .helper import validate_record
+goals_bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+# CREATE goal
+@goals_bp.route("", methods=["POST"])
+def create_goal():
+    request_body = request.get_json()
+
+    try:
+        new_goal = Goal.create(request_body)
+    except KeyError:
+        return abort(make_response(jsonify({"details":"Invalid data"}), 400))
+        
+    db.session.add(new_goal)
+    db.session.commit()
+
+    response_body = {}
+    response_body['goal'] = new_goal.to_json()
```

Comment:
We could combine these:

```suggestion
    response_body = {"goal": new_goal.to_json()}
```

Reply:
Great suggestion! I like it: just creating new dictionary with key:value pair.